### DISCUSSION TITLE
Windows changed bySmaller or Larger command snap back to previous size when dragging window around

### DISCUSSION THREAD

**Discussion by olliepop777** [rxhanson/Rectangle#1001] [General]
macOS version: 13.0
Rectangle version: 0.61

If you make a window smaller or larger using the respective commands, things are fine. But if you drag the window around after doing this, it will snap back to the original size before you did the smaller or larger command, as soon as you start moving the window around.

**Comment by rxhanson** [comment]
Thanks for reporting, sorry for the delayed response. The app currently treats all commands as though they are "snapped", which is misleading. You can uncheck the box for "Restore window size when unsnapped", but that would affect all commands. Since there is ambiguity here, I'll give it some thought and see what the best path might be - probably to change just smaller/larger to not be unsnap restored, but allow a terminal command configuration to reverse this since it's how the app has operated for years.

**Reply by olliepop777** [reply]
Ok, sorry I am new to this app and didn't realize this was a feature. It would be cool if they can be separated, but I understand this is different from how the app has behaved normally. Really loving the app though, and thanks for taking the time to respond!
